x-pack/plugins/screenshotting/server/screenshots/observable.ts

```diff
@@ -134,7 +153,11 @@ export class ScreenshotObservableHandler {
         timeoutWith(
           timeoutValue,
           throwError(
-            new Error(`The "${label}" phase took longer than ${timeoutValue / 1000} seconds.`)
+            new Error(
+              `Screenshotting encountered a timeout error: "${label}" took longer than` +
```

Comment:
This error is not translated, not sure if we actually want to translate it for something that rare.

Reply:
This error gets logged in the server logs and written into the report job metadata. When it is shown in the UI, it is nested in HTML `pre > code` tags. In general, I don't think it qualifies for translation.